src/openforms/formio/api/validators.py

```diff
@@ -56,7 +56,8 @@ def __init__(self, allowed_mime_types: Iterable[str] | None = None):
 
     def __call__(self, value: UploadedFile) -> None:
         head = value.read(2048)
-        ext = value.name.split(".")[-1]
+        file_name_parts = value.name.split(".")
```

Comment:
this way of grabbing the extension is actually quite flawed (it already was): try for example a file name `foo/bar.baz/example.png` :)

Instead, what is more correct to use is `pathlib`:

```py
from pathlib import Path

ext = Path(value.name).suffix[1:]
```
which gives you the actual extension (without dot)